### PR TITLE
Update CI with libproj 9.2.1 containers

### PR DESCRIPTION
**Pull request by urschrei**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

**Comment by urschrei** [comment]
@michaelkirk would you prefer to merge https://github.com/georust/docker-images/pull/25 prior to regenerating 1.63? (I thought it would include libproj 9.2.1, but I may be confused, as can so often be the way when Docker is involved)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -69,8 +69,8 @@ jobs:
           # Minimum supported rust version (MSRV)
           - "georust/geo-ci:rust-1.63"
```
Comment (michaelkirk) [inline .github/workflows/test.yml:70]:
```suggestion
          - "georust/geo-ci:proj-9.2.1-rust-1.63"
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -97,8 +97,8 @@ jobs:
...
```
Comment (michaelkirk) [inline .github/workflows/test.yml:98]:
```suggestion
          - "georust/geo-ci:proj-9.2.1-rust-1.63"
```

**Review by michaelkirk — APPROVED** [review]
I didn't clobber the existing tags with the new image. Maybe I should have?

In any case, I think if you reference the tags as in my suggestion things will "just work".

**Comment by urschrei** [comment]
> I didn't clobber the existing tags with the new image. Maybe I should have?

I'm not sure! It's a chicken and egg situation I think.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -124,13 +124,13 @@ jobs:
...
```
Comment (michaelkirk) [inline .github/workflows/test.yml:125]:
```suggestion
          - "georust/geo-ci:proj-9.2.1-rust-1.63"
```

Comment (michaelkirk) [reply .github/workflows/test.yml:125]: Sorry, missed one.

**Comment by urschrei** [comment]
There is something weird going on here. Gonna close this PR and try again.

**Pull request closed without merging**